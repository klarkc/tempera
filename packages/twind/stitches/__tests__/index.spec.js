import getStitches from "../src";

const tokens = {
  ComponentButtonBaseFontSize: '1rem',
  ComponentButtonBaseColor: 'red',
  ComponentButtonSmallFontSize: '0.75rem',
  ComponentButtonSecondaryDefaultBackgroundColor: 'red',
  ComponentButtonSecondaryDefaultColor: 'white',
  ComponentButtonSecondaryDefaultFontSize: '0.875rem',
  ComponentButtonSecondaryHoverFontSize: '0.75rem',
  ColorPrimary: 'red',
  ColorWhite: 'white',
  FontSizeTiny: '0.75rem',
  FontSizeSmall: '0.875rem',
  FontSizeMedium: '1rem',
  LineHeightTiny: '1rem',
  LineHeightSmall: '1.25rem',
  LineHeightMedium: '1.5rem'
};

describe("getStitches", () => {
  it("matches the saved snapshot", () => {
    const actual = getStitches(tokens);

    expect(actual).toMatchSnapshot();
  });
});
