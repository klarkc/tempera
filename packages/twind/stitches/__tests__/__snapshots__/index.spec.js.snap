// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getStitches matches the saved snapshot 1`] = `
Object {
  "button": Object {
    "base": "text-medium text-primary",
    "variants": Object {
      "size": Object {
        "small": "text-tiny",
      },
      "variant": Object {
        "secondary": "bg-primary text-white text-small hover:text-tiny",
      },
    },
  },
}
`;
